packages/MonorepoBuilder/tests/ComposerJsonDecorator/AutoloadRelativePathComposerJsonDecorator/AutoloadRelativePathComposerJsonDecoratorTest.php

```diff
@@ -51,14 +81,37 @@ private function getExpectedComposerJson(): array
         return [
             'autoload' => [
                 'psr-4' => [
-                    'App\\' => $this->getRelativeSourcePath() . '/src',
+                    'App\\' => $this->getRelativeSourcePath() . '/Source/src',
```

Comment:
Why such a change?

Here the `Source` directory should be in the `$this->getRelativeSourcePath()` method. It does something different then the name tells.